JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/timeseries/TimeSeriesClient.java

```diff
@@ -262,9 +263,11 @@ else if(type.get(i).equals(Type.GENERAL)){
    	    			throw new JPSRuntimeException(exceptionPrefix + "Inconsistent state created when initialising time series " + tsIRIs.get(i) +
    							" , as database related instantiation failed but KG triples were created.");
    	    		}
-   	    		throw new JPSRuntimeException(exceptionPrefix + "Timeseries was not created!", eRdbCreate);
+				failedTSIris.add(tsIRIs.get(i));
    	    	}
-   		}	
+   		}
+		if (!failedTSIris.isEmpty())
+			throw new JPSRuntimeException(exceptionPrefix + "Timeseries was not created!");
```

Comment:
The failed dataIRIs should be included in the error message, as time series IRIs are hidden from users

Reply:
resolved